New docs structure (#56)
opengisch/QField-docs: en/field-work/projects.rst

Open a project
##############
To open a project, tap on **Menu button --> Option button --> Open Project**.

Select the project you want to open by navigating to it, or use the **Recent projects** and **Favorites directories** shortcuts.

 .. image:: ../images/user-guide_open-project.png
    
     
 .. image:: ../images/user-guide_open-project-menu.png

Favorite directories
====================
In the first screen of the file selector, there is a section showing the favorite directories. To add a directory to the favorites, long click in the file selector on the directory name. To remove an entry from the favorites, long click on the entry in the favorites list.

Advanced project selection
==========================

> QField opens QGIS projects in the ".qgs" format. Currently it is not possible to open ".qgz" files.

Reviewer: ```suggestion
QField opens QGIS projects in the ".qgs" or ".qgz" format.
```

QField uses a customized file selector to open the projects. On choosing "Open project" from the QField menu, the file selector will be opened showing the following locations:

- The primary media/shared storage directory.
- The application-specific directories on all shared/external storage devices where the application can place persistent files it owns if they are not already included in the primary storage directory.

> Inside these directories, only subdirectories and files with the ".qgs" extension are shown.

Reviewer: ```suggestion
Inside these directories, only subdirectories and files with the ".qgs" or ".qgz" extension are shown.
```
